### PR TITLE
make weakly stable builds disableable

### PR DESCRIPTION
**Pull request by mikedanese**
cc @eparis @lavalamp

### CODE REVIEW

**Comment by eparis** [comment]
should we do the same for the 'strong' stable builds?  (less likely to be 'empty')

**Comment by eparis** [comment]
but this UI for the next person who comes along to have to remember one place to set the weak stable and another place to set the strong stable jobs is off-putting.

**Comment by eparis** [comment]
also, FYI, I looked at doing something like this a while ago, but never finished doing it...
https://github.com/kubernetes/contrib/compare/master...eparis:zero-empty-string-slices

**Comment by lavalamp** [comment]
I'm confused, doesn't just passing an empty string disable it? or a ","?

File: mungegithub/submit-queue/deployment.yaml [hunk 1]
```diff
@@ -16,6 +16,7 @@ spec:
         - --token-file=/etc/secret-volume/token
         - --pr-mungers=blunderbuss,lgtm-after-commit,cherrypick-auto-approve,label-unapproved-picks,needs-rebase,ok-to-test,rebuild-request,path-label,size,stale-pending-ci,stale-green-ci,block-path,release-note-label,comment-deleter,submit-queue
```
Comment (lavalamp) [inline mungegithub/submit-queue/deployment.yaml]: Oh, I'm a bad person and I changed the list in prod without checking it in. If we're keeping this file up to date, can you add issue-cache and flake-manager?

**Comment by lavalamp** [comment]
OK. Merging.

**Pull request merged**